### PR TITLE
Some fixes in russian translate

### PR DESCRIPTION
**Pull request by kinmoza**

### CODE REVIEW

File: appendix.ru.md [hunk 1]
```diff
@@ -2,38 +2,38 @@
...
+Взгляды Ричарда М. Столлмана на законы об изнасиловании и педофилии были публично обсуждены осенью 2019 года, когда о них написал Селам Ж. [1] Особенно пугает то, как Столлман относится к обвинениям в том, что Марвин Мински изнасиловал одну из жертв торговли людьми Джеффри Эпштейна (Вирджиния Джуффре), сказав: «но наиболее вероятный сценарий событий состоит в том, что она представила себя полностью готовой к интимным связям»  [2]. Ричард М. Столлман заявляет, что это не «сексуальное насилие», потому что «нападение предполагает, что нападающий применил силу или принуждение», в то время как в обсуждаемом отчете «нет ничего подобного. Только то, что у них был секс.». [3] Вместо того, чтобы обсуждать это дальше, лучше сосредоточиться на его личном веб-сайте, где он делится своими взглядами на то, что несовершеннолетние «полностью готовы к интимным отношениям». [4]. (Примечание. Хотя несколько новостных сообщений и искажали позицию Ричарда М. Столлмана на тему обсуждения обвинений против Мински, Ричард М. Столлман ранее сам высказывал мнение, которое полностью соответствовало искаженному.)
```
Comment (andrewshadura) [inline appendix.ru.md:9]: Selam is her, not him. When not taking my changes, maybe it’s worth at least reading them before carrying on and ignoring them.

Comment (andrewshadura) [reply appendix.ru.md:9]: Same about Guiffre, Minsky, you didn’t take my changes.

File: appendix.ru.md [hunk 2]
```diff
@@ -2,38 +2,38 @@
...
 [3]: https://web.archive.org/web/20210325013629/https://www.vice.com/en/article/9ke3ke/famed-computer-scientist-richard-stallman-described-epstein-victims-as-entirely-willing
 [4]: https://web.archive.org/web/20210325013706/https://stallman.org/archives/2018-jul-oct.html#23_September_2018_(Cody_Wilson)
 
-Он неоднократно комментировал «Нечестный закон, который, квалифицирует секс с подростками как «изнасилование», даже если они хотят этого». [5] Он сравнивает законы США с Суданскими законами, говоря, что «законы США, определяющие изнасилование, так же включают добровольное секс с детьми младше N лет (где N - варьируется)» и что « оба закона искажают значение слова изнасилование» [6].
+Он неоднократно комментировал «нечестный закон, который, квалифицирует секс с подростками как "изнасилование", даже если они хотят этого». [5] Он сравнивает законы США с Суданскими законами, говоря, что «законы США, определяющие изнасилование, так же включают добровольное секс с детьми младше N лет (где N - варьируется)» и что «оба закона искажают значение слова изнасилование» [6].
```
Comment (andrewshadura) [inline appendix.ru.md:16]: Here, you changed the correct quotation marks I used to the incorrect ones (`"`).

File: appendix.ru.md [hunk 3]
```diff
@@ -2,38 +2,38 @@
...
+Говоря о женщине, которая занималась сексом с несовершеннолетним, он высказал: «В свои 14 я мечтал, чтобы привлекательная дама меня "изнасиловала"». [7] Что касается детской порнографии, он высказался прямо: «производство фотографий такого рода - преступление, но это не повод запрещать их хранение» [8]. В целом он защищал педофилию, утверждая, что «мало доказательств, подтверждающих широко распространённое предположение о том, что добровольное участие в акте педофилии вредит детям». [9] (Примечание. : Ричард М. Столлман принёс свои извинения за этот комментарий 14 сентября 2019 г. Примечание #2: ссылка добавлена 24 марта 2021 г [Ссылка][10]. - 08:50 EDT.»)
```
Comment (andrewshadura) [inline appendix.ru.md:21]: `высказал` is not valid Russian.
Ditto re quotation marks.

File: appendix.ru.md [hunk 4]
```diff
@@ -2,38 +2,38 @@
...
+Ричард М. Столлман потратил годы на кампанию против использования правильных местоимений. Это плохо замаскированная трансфобия. В оригинальной публикации GNU Kind Communication Guidelines он сказал, что «существуют различные способы выражения гендерной нейтральности в местоимениях единственного числа от третьего лица в английском языке; вам не обязательно использовать местоимение "они" ("they")» [15]. С тех пор этот текст был обновлен, но он все еще трансфобен. [16] На главной странице его веб-сайта есть утверждение, что «они» имеет множественное число и что для отдельных лиц следует использовать «гендерно-нейтральные местоимения». [17]
```
Comment (andrewshadura) [inline appendix.ru.md:36]: Ditto.

File: index.ru.md [hunk 5]
```diff
@@ -11,7 +11,7 @@ lang: ru
...
 
-Мы призываем к отстранению всех членов совета правления FSF. Это люди, которые поддерживали полномочия Ричарда М. Столлмана в течение многих лет. Они еще раз демонстрируют это, разрешая ему вернуться в Правление FSF. Пришло время RMS отойти от свободного программного обеспечения, технической этики, цифровых прав и сообществ, поскольку он не может обеспечить лидерство, в котором мы нуждаемся. Мы также призываем к отстранению Ричарда М. Столлмана от всех других руководящих должностей, включая проект GNU.
+Мы призываем к отстранению всех членов совета Правления FSF. Это люди, которые поддерживали полномочия Ричарда М. Столлмана в течение многих лет. Они еще раз демонстрируют это, разрешая ему вернуться в Правление FSF. Пришло время RMS отойти от свободного программного обеспечения, технической этики, цифровых прав и сообществ, поскольку он не может обеспечить лидерство, в котором мы нуждаемся. Мы также призываем к отстранению Ричарда М. Столлмана от всех других руководящих должностей, включая проект GNU.
```
Comment (andrewshadura) [inline index.ru.md:14]: `Правление` is not the correct term, please find the correct one in the second paragraph.

**Review by andrewshadura — CHANGES_REQUESTED** [review]
Please take changes from PR #2543.

File: appendix.ru.md (hunk 3 above)
Comment (kinmoza) [reply appendix.ru.md:21]: "Высказал" очень даже корректное слово.
https://ru.wiktionary.org/wiki/%D0%B2%D1%8B%D1%81%D0%BA%D0%B0%D0%B7%D0%B0%D1%82%D1%8C

Comment (andrewshadura) [reply appendix.ru.md:21]: 1. Please reply in English.
2. You can only высказать something, you cannot just высказать without an object. A citation is not an object. You need to use высказался or сказал.

Comment (isagalaev) [reply appendix.ru.md:21]: Although "высказал" is a valid word in itself I agree it's wrong to use it here. It's simply "сказал".

File: appendix.ru.md [hunk 6]
```diff
@@ -2,38 +2,38 @@
...
+В 2015 и 2016 годах Ричард М. Столлман разместил на своём сайте три сообщения по поводу синдрома Дауна. Он рекомендовал узнавшим о положительном результате теста на синдром Дауна у плода прерывание беременности в качестве правильного решения. [11]. В своём заявлении он ссылался на людей, сказавших, что «нет ничего хорошего в увеличении количества людей с синдромом Дауна». [12]. Так же он заявлял, что «когда ребёнок болен синдромом Дауна, лучше всего прервать беременность и попробовать зачать снова». [13] По крайней мере в одном случае, Ричард М. Столлман сравнил наличие в семье ребенка с синдромом Дауна с домашним животным [14] (Примечание: ссылка 14 добавлена 25 марта 2021 г. - 11:33 EDT).
```
Comment (isagalaev) [inline appendix.ru.md:28]: Лишняя запятая после "в одном случае" (это простое обстоятельство времени, оно не выделяется запятыми).

File: appendix.ru.md (hunk 1 above)
Comment (andrewshadura) [reply appendix.ru.md:9]: Giuffre = Джюффре, not Джуффре
Minsky = Минский, not Мински

File: index.ru.md (hunk 5 above)
Comment (isagalaev) [inline index.ru.md:14]: +1

File: appendix.ru.md [hunk 7]
```diff
@@ -2,38 +2,38 @@
 lang: ru
 ---
 
-Ричард М. Столлман известен своим негативным отношением к женщинами, заставляя их чувствоваться себя некомфортно и уязвимо. Информацию об инцидентах, связанных с Ричардом М. Столлманом и MIT, см. в [0].
+Ричард М. Столлман, известный своим негативным отношением к женщинами, заставлял их чувствовать себя некомфортно и уязвимо. Информацию об инцидентах, связанных с Ричардом М. Столлманом и MIT, см. в [0].
```
[the remark this answers is not in the document]
Comment (andrewshadura) [reply appendix.ru.md:5]: The original reads weird, the changed one is slightly better.

File: appendix.ru.md [hunk 8]
```diff
@@ -6,26 +6,26 @@ lang: ru
...
 [3]: https://web.archive.org/web/20210325013629/https://www.vice.com/en/article/9ke3ke/famed-computer-scientist-richard-stallman-described-epstein-victims-as-entirely-willing
 [4]: https://web.archive.org/web/20210325013706/https://stallman.org/archives/2018-jul-oct.html#23_September_2018_(Cody_Wilson)
 
-Он неоднократно комментировал «нечестный закон, который, квалифицирует секс с подростками как "изнасилование", даже если они хотят этого». [5] Он сравнивает законы США с Суданскими законами, говоря, что «законы США, определяющие изнасилование, так же включают добровольный секс с детьми младше N лет (где N - варьируется)» и что «оба закона искажают значение слова изнасилование» [6].
+Он неоднократно комментировал «нечестный закон, который квалифицирует секс с подростками как 'изнасилование', даже если они хотят этого». [5] Он сравнивает законы США с Суданскими законами, говоря, что «законы США, определяющие изнасилование, так же включают добровольный секс с детьми младше N лет (где N - варьируется)» и что «оба закона искажают значение слова изнасилование» [6].
```
Comment (andrewshadura) [inline appendix.ru.md:16]: „“

File: appendix.ru.md [hunk 9]
```diff
@@ -6,26 +6,26 @@ lang: ru
...
+Взгляды Ричарда М. Столлмана на законы об изнасиловании и педофилии были публично обсуждены осенью 2019 года, когда о них написала Селам Ж. [1] Особенно пугает то, как Столлман относится к обвинениям в том, что Марвин Минский изнасиловал одну из жертв торговли людьми Джеффри Эпштейна (Вирджиния Джюффре), сказав: «но наиболее вероятный сценарий событий состоит в том, что она представила себя полностью готовой к интимным связям»  [2]. Ричард М. Столлман заявляет, что это не «сексуальное насилие», потому что «нападение предполагает, что нападающий применил силу или принуждение», в то время как в обсуждаемом отчете «нет ничего подобного. Только то, что у них был секс.». [3] Вместо того, чтобы обсуждать это дальше, лучше сосредоточиться на его личном веб-сайте, где он делится своими взглядами на то, что несовершеннолетние «полностью готовы к интимным отношениям». [4]. (Примечание. Хотя несколько новостных сообщений и искажали позицию Ричарда М. Столлмана на тему обсуждения обвинений против Марвина, Ричард М. Столлман ранее сам высказывал мнение, которое полностью соответствовало искаженному.)
```
Comment (andrewshadura) [inline appendix.ru.md:9]:
```suggestion
Взгляды Ричарда М. Столлмана на законы об изнасиловании и педофилии были публично обсуждены осенью 2019 года, когда о них написала Селам Ж. [1] Особенно пугает то, как Столлман относится к обвинениям в том, что Марвин Минский изнасиловал одну из жертв торговли людьми Джеффри Эпштейна (Вирджиния Джюффре), сказав: «но наиболее вероятный сценарий событий состоит в том, что она представила себя полностью готовой к интимным связям»  [2]. Ричард М. Столлман заявляет, что это не «сексуальное насилие», потому что «нападение предполагает, что нападающий применил силу или принуждение», в то время как в обсуждаемом отчете «нет ничего подобного. Только то, что у них был секс.» [3] Вместо того, чтобы обсуждать это дальше, лучше сосредоточиться на его личном веб-сайте, где он делится своими взглядами на то, что несовершеннолетние «полностью готовы к интимным отношениям». [4]. (Примечание. Хотя несколько новостных сообщений и искажали позицию Ричарда М. Столлмана на тему обсуждения обвинений против Марвина, Ричард М. Столлман ранее сам высказывал мнение, которое полностью соответствовало искаженному.)
```

File: appendix.ru.md (hunk 8 above)
Comment (andrewshadura) [reply appendix.ru.md:16]:
```suggestion
Он неоднократно комментировал «нечестный закон, который квалифицирует секс с подростками как „изнасилование“, даже если они хотят этого». [5] Он сравнивает законы США с Суданскими законами, говоря, что «законы США, определяющие изнасилование, так же включают добровольный секс с детьми младше N лет (где N - варьируется)» и что «оба закона искажают значение слова изнасилование» [6].
```

File: appendix.ru.md [hunk 10]
```diff
@@ -6,26 +6,26 @@ lang: ru
...
+Говоря о женщине, которая занималась сексом с несовершеннолетним, он сказал: «В свои 14 я мечтал, чтобы привлекательная дама меня 'изнасиловала'». [7] Что касается детской порнографии, он высказался прямо: «производство фотографий такого рода - преступление, но это не повод запрещать их хранение» [8]. В целом он защищал педофилию, утверждая, что «мало доказательств, подтверждающих широко распространённое предположение о том, что добровольное участие в акте педофилии вредит детям». [9] (Примечание. : Ричард М. Столлман принёс свои извинения за этот комментарий 14 сентября 2019 г. Примечание #2: ссылка добавлена 24 марта 2021 г [Ссылка][10]. - 08:50 EDT.»)
```
Comment (andrewshadura) [inline appendix.ru.md:21]:
```suggestion
Говоря о женщине, которая занималась сексом с несовершеннолетним, он сказал: «В свои 14 я мечтал, чтобы привлекательная дама меня 'изнасиловала'». [7] Что касается детской порнографии, он высказался прямо: «производство фотографий такого рода - преступление, но это не повод запрещать их хранение» [8]. В целом он защищал педофилию, утверждая, что «мало доказательств, подтверждающих широко распространённое предположение о том, что добровольное участие в акте педофилии вредит детям». [9] (Примечание: Ричард М. Столлман принёс свои извинения за этот комментарий 14 сентября 2019 г. Примечание #2: ссылка добавлена 24 марта 2021 г [Ссылка][10]. - 08:50 EDT.»)
```

**Comment by kinmoza** [comment]
Check it now

**Review by andrewshadura — CHANGES_REQUESTED** [review]

File: appendix.ru.md (hunk 8 above)
Comment (kinmoza) [reply appendix.ru.md:16]: In original text `''`

File: appendix.ru.md (hunk 9 above)
Comment (kinmoza) [reply appendix.ru.md:9]: Already fixed

File: appendix.ru.md (hunk 8 above)
Comment (andrewshadura) [reply appendix.ru.md:16]: I doesn’t matter. In Russian, nested quotation marks are «„“».

File: appendix.ru.md (hunk 9 above)
Comment (andrewshadura) [reply appendix.ru.md:9]: No, it isn’t:
Screenshot from 2021-03-31 19-21-08

(The change is about removing the second full stop.)

Comment (andrewshadura) [reply appendix.ru.md:9]: It is now.

Comment (kinmoza) [reply appendix.ru.md:9]: Sorry, i'll remove this dot.

File: appendix.ru.md (hunk 8 above)
Comment (kinmoza) [reply appendix.ru.md:16]: Okay.

File: appendix.ru.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (isagalaev) [reply appendix.ru.md:5]: They frankly read both awkward to me, but I couldn't come up with a better one myself :-)

File: appendix.ru.md (hunk 8 above)
Comment (isagalaev) [reply appendix.ru.md:16]: Just to note my voice: typographical quotes are not my strong suite, so on those points I defer to Andrew who seems to know them better.

File: appendix.ru.md [hunk 11]
```diff
@@ -2,38 +2,38 @@
...
 [3]: https://web.archive.org/web/20210325013629/https://www.vice.com/en/article/9ke3ke/famed-computer-scientist-richard-stallman-described-epstein-victims-as-entirely-willing
 [4]: https://web.archive.org/web/20210325013706/https://stallman.org/archives/2018-jul-oct.html#23_September_2018_(Cody_Wilson)
 
-Он неоднократно комментировал «Нечестный закон, который, квалифицирует секс с подростками как «изнасилование», даже если они хотят этого». [5] Он сравнивает законы США с Суданскими законами, говоря, что «законы США, определяющие изнасилование, так же включают добровольное секс с детьми младше N лет (где N - варьируется)» и что « оба закона искажают значение слова изнасилование» [6].
+Он неоднократно комментировал «нечестный закон, который, квалифицирует секс с подростками как "изнасилование", даже если они хотят этого». [5] Он сравнивает законы США с Суданскими законами, говоря, что «законы США, определяющие изнасилование, так же включают добровольный секс с детьми младше N лет (где N - варьируется)» и что «оба закона искажают значение слова изнасилование» [6].
```
[the remark this answers is not in the document]
Comment (kinmoza) [reply appendix.ru.md]: Fixed.

File: appendix.ru.md (hunk 6 above)
Comment (kinmoza) [reply appendix.ru.md]: Fixed.

File: index.ru.md (hunk 5 above)
Comment (kinmoza) [reply index.ru.md]: Fixed.

File: appendix.ru.md (hunk 8 above)
Comment (kinmoza) [reply appendix.ru.md]: Fixed.

File: appendix.ru.md (hunk 9 above)
Comment (kinmoza) [reply appendix.ru.md]: Fixed.

File: appendix.ru.md (hunk 1 above)
Comment (kinmoza) [reply appendix.ru.md]: Fixed?

**Comment by kinmoza** [comment]
So... Are there any more edits or comments?

**Comment by ehashman** [comment]
Thanks everyone!

**Comment by kinmoza** [comment]
Good job! Thanks for your suggestions!

**Pull request merged**